.github/CODEOWNERS

```
@@ -30,7 +30,7 @@ satpy/etc/readers/avhrr_l1b_gaclac.yaml @mraspaud @sfinkens
 satpy/etc/readers/avhrr_l1b_hrpt.yaml @mraspaud
 satpy/etc/readers/clavrx.yaml @djhoese
 satpy/etc/readers/electrol_hrit.yaml @sfinkens @mraspaud
-satpy/etc/readers/fci_l1c_fdhsi.yaml @mraspaud
+satpy/etc/readers/fci_l1c_nc.yaml @mraspaud
```

you and @gerritholl have worked more on this reader than me, maybe you should be the new code owners?
```suggestion
satpy/etc/readers/fci_l1c_nc.yaml @ameraner @gerritholl
```

---

Thanks for offering! I can gladly be codeowner. @gerritholl do you agree as well?

---

Yes, works for me.  Now I know how GitHub figures out the code owners :)